plugins/cloud-gce/src/test/java/org/elasticsearch/discovery/gce/GceComputeServiceMock.java

```diff
@@ -91,23 +90,28 @@ private String readGoogleApiJsonResponse(String url) throws IOException {
 
     private String readJsonResponse(String url, String urlRoot) throws IOException {
         // We extract from the url the mock file path we want to use
-        String mockFileName = Strings.replace(url, urlRoot, "") + ".json";
+        String mockFileName = Strings.replace(url, urlRoot, "");
 
         logger.debug("--> read mock file from [{}]", mockFileName);
-        URL resource = GceComputeServiceMock.class.getResource(mockFileName);
+        URL resource = GceComputeServiceMock.class.getResource(mockFileName + ".json");
```

Comment:
agreed. Well actually I don't need any extension so it will look even closer to what GCE API does.
I did that at the first place so the IDE will be able to propose syntax highlighting. 
But I agree, let's remove that.